server/deskroom/common/openai.py

```diff
@@ -1,11 +1,12 @@
+import os
 from pathlib import Path
 
 from openai import AsyncAzureOpenAI, AsyncOpenAI, AzureOpenAI, OpenAI
 
 from deskroom.config import settings
 from deskroom.constants import PROMPT_PATH
 
-from .azure import create_azure_container_client
+from .azure import create_azure_service_client
```

Comment:
relative import는 최대한 지양하는 게 좋을거 같습니다. 
```suggestion
from deskroom.common.azure import create_azure_service_client
```